requirements.txt

```diff
@@ -11,3 +11,4 @@ PyQt6-Qt6~=6.5.3; sys_platform == 'darwin'
 python-dateutil~=2.7
 pywin32; sys_platform == 'win32'
 PyYAML>=5.1, <7
+chardet~=5.2.0
```

Comment:
This extra dependency bothers me a bit, especially to solve such a corner case.

Reply:
We did do the BOM detection to avoid extra dependency, and because all the examples we had where using some kind of Unicode encoding. But if we need to assume that essentially any text encoding is possible and we need to detect it I don't think we get around adding an additional dependency. The only alternative would be to reimplement chardet's functionality, and that is far worse.

Reply:
> that i18n is a corner case.

Not sure what you are talking about, this isn't about i18n but detection of historical charsets used by externally generated files that Picard may need to read, this PR is about solving a very specific use case.
This detection is only a guess, and may not work in all cases.

Unicode is around since more than 30 years.

> But if we need to assume that essentially any text encoding is possible and we need to detect it I don't think we get around adding an additional dependency. The only alternative would be to reimplement chardet's functionality, and that is far worse.

To me, the question is:
should we add support for that  (and the best way is clearly `chardet`) or just decide Picard doesn't support them and let user convert his files to Unicode before trying to load them in Picard.

Not a big deal here, I'm ok to support such files whenever possible but better think twice before adding any extra dependency, as we know it will always translate to extra work for us at some point (and it starts here).